doc/source/user_guide_scripting/helper_scripts.rst

```diff
@@ -0,0 +1,342 @@
+- :ansyshelp:`Add voltage ground <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Add voltage <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Add plastic heating <act_script_demo_coupled_field_transient.html>`
+
+Solution setup
+--------------
+
+- :ansyshelp:`Set transient analysis settings containing multiple steps <act_script_demo_trans_struct.html>`
+- :ansyshelp:`Set modal analysis settings <act_script_demo_coupled_field_modal.html>`
+- :ansyshelp:`Set random vibration analysis settings <act_script_demo_random_vib.html>`
+- :ansyshelp:`Set modal acoustic analysis settings <act_script_demo_modal_acoustic.html>`
+- :ansyshelp:`Set harmonic acoustic analysis settings <act_script_demo_harmonic_acoustic.html>`
+- :ansyshelp:`Set transient thermal analysis settings <act_script_demo_transient_therm.html>`
+- :ansyshelp:`Set steady state thermal analysis settings <act_script_demo_steady_state_therm.html>`
+- :ansyshelp:`Set RBD analysis settings <act_script_demo_rbd_flexible.html>`
+- :ansyshelp:`Set electric analysis settings <act_script_demo_electric.html>`
+
+------------------------
+
+- :ansyshelp:`Set the harmonic range maximum and solution intervals <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Set the modal max modes to find and search the range <act_script_demo_coupled_field_modal.html>`
+
+------------------------
+
+- :ansyshelp:`Set convergence settings <act_script_demo_coupled_field_transient.html>`
+- :ansyshelp:`Set sub-steps <act_script_demo_steady_state_therm.html>`
+- :ansyshelp:`Set the step end time <act_script_demo_Cyclic.html>`
+- :ansyshelp:`Set automatic time stepping <act_script_demo_cylindrical_joint.html>`
+- :ansyshelp:`Set the solver type <act_script_demo_coupled_field_modal.html>`
+- :ansyshelp:`Perform a solution while specifying the solution handler and the number of cores <act_script_examples_solve_track_core.html>`
+- :ansyshelp:`Solve <act_script_demo_static_struct.html>`
+
+Result post-processing
+----------------------
+
+- :ansyshelp:`Add directional deformation <act_script_demo_general_joint.html>`
+- :ansyshelp:`Add directional deformation (scope to named selection) <act_script_demo_static_struct.html>`
+- :ansyshelp:`Get the maximum or minimum value of a result <act_script_demo_static_struct.html>`
+- :ansyshelp:`Add equivalent stress <act_script_demo_trans_struct.html>`
+- :ansyshelp:`Add normal stress <act_script_demo_coupled_field_001.html>`
+- :ansyshelp:`Add thermal strain <act_script_demo_coupled_field_001.html>`
+- :ansyshelp:`Add middle principal elastic strain <act_script_demo_rbd_flexible.html>`
+- :ansyshelp:`Add equivalent plastic strain <act_script_demo_coupled_field_transient.html>`
+- :ansyshelp:`Add normal elastic strain <act_script_demo_random_vib.html>`
+- :ansyshelp:`Add a stress tool <act_script_demo_trans_struct.html>`
+- :ansyshelp:`Add a user defined result <act_script_demo_post2.html>`
```

Comment:
user-defined result should have a hyphen, but again, I assume you are matching the title in the help.